core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -285,12 +288,17 @@ private List<List<KafkaConnectorModelResponse>> getPagedList(
   }
 
   private void updateTeamNamesForDisplay(List<KafkaConnectorModelResponse> topicListUpdated) {
-    topicListUpdated.forEach(
-        topicInfo -> {
-          if (topicInfo.getTeamName().length() > 9) {
-            topicInfo.setTeamName(topicInfo.getTeamName().substring(0, 8) + "...");
-          }
-        });
+    topicListUpdated.stream()
+        .filter(
+            t ->
+                t.getTeamName().length() > TOPIC_NAME_MAX_LENGTH - ELLIPSIS.length() + 1
+                    && (t.getTeamName().length() != TOPIC_NAME_MAX_LENGTH
+                        || !t.getTeamName().endsWith(ELLIPSIS)))
+        .forEach(
+            topicInfo ->
+                topicInfo.setTeamName(
+                    topicInfo.getTeamName().substring(0, TOPIC_NAME_MAX_LENGTH - ELLIPSIS.length())
```

Comment:
there is no unit test for this method previously. possible to add one ?

Reply:
test added